src/util/simplify_expr2.cpp

```diff
@@ -133,8 +135,8 @@ static void fetch_ops_from_this_type(
 {
   if(expr->expr_id == id)
   {
-    expr->foreach_operand(
-      [&ops, id](const expr2tc &e) { fetch_ops_from_this_type(ops, id, e); });
+    expr->foreach_operand([&ops, id](const expr2tc &e)
+                          { fetch_ops_from_this_type(ops, id, e); });
```

Comment:
Why are you changing the original formating?

Reply:
I believe this is due to a change in `clang-format` between 12 and 13 affecting (at least) nested block formatting.